tests/dummy/app/elm-modules/Chat.elm

```diff
@@ -1,76 +1,97 @@
 port module Chat exposing (main)
 
+import Browser
 import Html exposing (..)
 import Html.Attributes exposing (..)
 import Html.Events exposing (..)
-import WebSocket as WS
```

Comment:
Websocket support is not in 0.19 yet, so my only option was to remove the import. Makes this program less interesting, but it still compiles.

Reply:
Looks good to me!